src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,56 +1,139 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+interface Props {
+  onAdd: (movieToAdd: Movie) => void;
+}
+
+const initialFormState = {
+  newTitle: '',
+  newDescription: '',
+  newImgUrl: '',
+  newImdbUrl: '',
+  newImdbId: '',
```

Comment:
You could just name it as in Movie type, so your initial state and new movie will match the type
```suggestion
  title: '',
  description: '',
  imgUrl: '',
  imdbUrl: '',
  imdbId: '',
```